client/src/hu_elements.cpp

```diff
@@ -451,7 +451,7 @@ std::string NetdemoElapsed() {
 	timeelapsed -= minutes * 60;
 	int seconds = timeelapsed;
 
-	char str[9];
+	char str[21];
```

Comment:
Any reason for 21 instead of 25 like the other lines above ?

Reply:
The compiler warns about the possibility of lacking space, there is a risk for segfault in the sprintf() functions right below. 9 bytes are sufficient, but gcc insists it isn't. 20 bytes still results in warning, 21 doesn't.

Reply:
For any other occurrences, I followed the same principle, that is, to raise the number of bytes for each array until gcc stopped issuing warnings, all of them were warnings around the usage of sprintf() in different places.